bc_obps/service/tests/test_operation_service.py

```diff
@@ -1061,6 +1067,30 @@ def test_raises_exception_if_no_documents():
         with pytest.raises(Exception, match="Operation must have a process flow diagram and a boundary map."):
             OperationService.raise_exception_if_operation_missing_registration_information(operation)
 
+    @staticmethod
+    def test_raises_exception_if_documents_are_quarantined():
+
+        operation = set_up_valid_mock_operation(
+            Operation.Purposes.OPTED_IN_OPERATION, document_scan_status="Quarantined"
+        )
+
+        with pytest.raises(
+            Exception,
+            match="Virus detected in test.pdf, test.pdf. Please go back and replace these attachments before submitting.",
+        ):
+            OperationService.raise_exception_if_operation_missing_registration_information(operation)
+
+    @staticmethod
+    def test_raises_exception_if_documents_are_still_unscanned():
+
+        operation = set_up_valid_mock_operation(Operation.Purposes.OPTED_IN_OPERATION, document_scan_status="Unscanned")
+
+        with pytest.raises(
+            Exception,
+            match="Please wait. Your attachments are being scanned for viruses, this may take a few minutes.",
```

Comment:
```suggestion
            match="Please wait. Your attachments are being scanned for malware, this may take a few minutes.",
```

Reply:
Reworded in [f2bcd2a](https://github.com/bcgov/cas-registration/pull/3137/commits/f2bcd2a0cb943ecf2255671544f098d98a762cd9)